app/screens/wallet/SendCoins.tsx

```diff
@@ -91,41 +91,44 @@ const SendCoins = ({ history, location }: Props) => {
   };
 
   const renderTxParamsMode = () => {
-    return [
-      <TxParams
-        fromAddress={currentAccount.publicKey}
-        address={address}
-        hasAddressError={hasAddressError}
-        updateTxAddress={updateTxAddress}
-        resetAddressError={() => setHasAddressError(false)}
-        amount={amount}
-        updateTxAmount={updateTxAmount}
-        hasAmountError={hasAmountError}
-        resetAmountError={() => setHasAmountError(false)}
-        updateFee={updateFee}
-        note={note}
-        updateTxNote={updateTxNote}
-        cancelTx={history.goBack}
-        nextAction={proceedToMode2}
-        contacts={contacts}
-        key="params"
-        isDarkMode={isDarkMode}
-      />,
-      isCreateNewContactOn ? (
-        <CreateNewContact
-          isStandalone
-          initialAddress={address}
-          onCompleteAction={() => setIsCreateNewContactOn(false)}
-          onCancel={() => setIsCreateNewContactOn(false)}
-          key="newContact"
+    return (
+      <>
+        <TxParams
+          fromAddress={currentAccount.publicKey}
+          address={address}
+          hasAddressError={hasAddressError}
+          updateTxAddress={updateTxAddress}
+          resetAddressError={() => setHasAddressError(false)}
+          amount={amount}
+          updateTxAmount={updateTxAmount}
+          hasAmountError={hasAmountError}
+          resetAmountError={() => setHasAmountError(false)}
+          updateFee={updateFee}
+          note={note}
+          updateTxNote={updateTxNote}
+          cancelTx={history.goBack}
+          nextAction={proceedToMode2}
+          contacts={contacts}
+          key="params"
+          isDarkMode={isDarkMode}
         />
-      ) : (
-        <TxSummary address={address} fromAddress={currentAccount.publicKey} amount={parseInt(`${amount}`)} fee={fee} note={note} key="summary" />
-      )
-    ];
+        {isCreateNewContactOn ? (
+          <CreateNewContact
+            isStandalone
+            initialAddress={address}
+            onCompleteAction={() => setIsCreateNewContactOn(false)}
+            onCancel={() => setIsCreateNewContactOn(false)}
+            key="newContact"
+          />
+        ) : (
+          <TxSummary address={address} fromAddress={currentAccount.publicKey} amount={parseInt(`${amount}`)} fee={fee} note={note} key="summary" />
+        )}
+      </>
+    );
   };
 
   switch (mode) {
+    default: // Hopefully it never defaults, but in case that this happened at least render a first step
```

Comment:
Do you think that is an empty screen is better than render something for such a case?
Moreover, `mode` has an explicit type `1 | 2 | 3` and mode handling implemented right in the component (no possible side-effects on this variable), so TypeScript should help us to avoid the case for `default`.
But it's better to have a `default` case for switches, much better than shut up eslint :)